File: php/3090 - Classifying Triangles II.php
<?php
$stdin = fopen('php://stdin', 'r');

$input = fgets($stdin, 10);
list($a, $b, $c) = split(" ",$input);

if ((($a*$a)==($b*$b)+($c*$c))||(($b*$b)==($a*$a)+($c*$c))||(($c*$c)==($a*$a)+($b*$b))) {
	echo "rectangulo \n";
} else if ((($a*$a)>($b*$b)+($c*$c))||(($b*$b)>($a*$a)+($c*$c))||(($c*$c)>($a*$a)+($b*$b))) {
	echo "obtusangulo \n";
} else {
	echo "acutangulo \n";
}
		
fclose($stdin);
?>

File: Java/3090 - Classifying Triangles II.java
import java.util.*;

public class coj {

	public static void main(String[] args) {
		int a, b, c;
        Scanner sc = new Scanner(System.in);
        a = sc.nextInt(); b = sc.nextInt(); c = sc.nextInt();
        
        if ((Math.pow(a, 2)==Math.pow(b, 2)+Math.pow(c, 2))||(Math.pow(b, 2)==Math.pow(a, 2)+Math.pow(c, 2))||(Math.pow(c, 2)==Math.pow(a, 2)+Math.pow(b, 2))) {
			System.out.println("rectangulo");
		} else if ((Math.pow(a, 2)>Math.pow(b, 2)+Math.pow(c, 2))||(Math.pow(b, 2)>Math.pow(a, 2)+Math.pow(c, 2))||(Math.pow(c, 2)>Math.pow(a, 2)+Math.pow(b, 2))) {
			System.out.println("obtusangulo");
		} else {
			System.out.println("acutangulo");
		}
	}

}